.github/workflows/shadow-repo.yml

```
@@ -0,0 +1,72 @@
+name: Sync with shadow repo
+
+on:
+  push:
+  create:
+  delete:
+  pull_request:
+    types: [opened]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+jobs:
+  sync-branch:
+    name: Create/delete shadow branch
+    runs-on: ubuntu-latest
+    if: github.event.ref_type == 'branch'
+    steps:
+      - name: Clone shadow repo
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ secrets.SHADOW_OWNER }}/${{ secrets.SHADOW_REPO }}
```

```suggestion
          repository: ${{ vars.SHADOW_OWNER }}/${{ vars.SHADOW_REPO }}
```

I think these two values can be moved to the Variables sections of the Settings. It makes debugging and understanding the logs a bit easier and their contents are displayed in the PR comments anyways